frontend/src/components/MainNavigation.vue

```diff
@@ -255,22 +288,33 @@ export default {
       return (route) => {
         return namespacedRoute(route, this.namespace)
       }
+    },
+    projectFilterHasExactMatch () {
+      const projectName = get(head(this.sortedAndFilteredProjectList), 'metadata.name')
```

Comment:
do not inline lodash functions if possible since we don't have chain.
```js
const project = head(this.sortedAndFilteredProjectList)
const projectName = get(project, 'metadata.name')
```

Reply:
done